packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -0,0 +1,265 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+
+export type ButtonProps = {
```

Comment:
Would be great if we can add JSdocs to the props so they are shown in Storybook

Reply:
I would need more information (or an example) about this

Reply:
Example:
```ts
export type OnyxButtonProps = {
  /**
   * Label to show.
   */
  label?: string;
};

```